lib/mix/tasks/scenic.gen.component.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Mix.Tasks.Scenic.Gen.Component do
+  @shortdoc "Creates scaffolding for a new Scenic component."
+
+  @moduledoc """
+  Generates Scene boilerplate for a new component.
+
+  ```bash
+  mix scenic.gen.component component_name
+  ```
+  """
+  use Mix.Task
+
+  import Mix.Generator
+  alias ScenicNew.Common
+
+  @switches [
+    app: :string,
+    module: :string
+  ]
+
+
+  # --------------------------------------------------------
+  @doc false
+  def run(argv) do
+    if Mix.Project.umbrella?() do
+      Mix.raise "mix scenic.gen.component must be invoked from within your scenic application root directory."
+    end
+
+    {opts, argv} = OptionParser.parse!(argv, strict: @switches)
+
+    case argv do
+      [] ->
+        Mix.Tasks.Help.run(["scenic.gen.component"])
+
+      [component_name | _] ->
+
+        Common.elixir_version_check!()
+        app = opts[:app] || Path.basename(Path.expand(component_name))
+        Common.check_application_name!(app, !opts[:app])
+        mod = opts[:module] || Macro.camelize(app)
```

Comment:
Please correct me if I'm wrong, but typically community is doing the opposite of that (`underscore` module) and that's because:
```elixir
iex> Macro.underscore("MyHTTP")
"my_http"
iex> Macro.camelize("my_http")
"MyHttp"
```
see:
https://hexdocs.pm/phoenix/Mix.Tasks.Phx.Gen.Channel.html#content
and other `mix` tasks in for this library.

Reply:
Oh, you mean, doing:

`mix gen.scenic.scene MyScene`

instead of 

`mix gen.scenic.scene my_scene`

?

Reply:
exactly

Reply:
Good idea, will change.

Reply:
Fixed!